client/src/components/Menu/Menu.js

```diff
@@ -36,26 +47,34 @@ class Menu extends Component {
                 <i className="fa fa-question-circle fa-fw" /> Help
               </Link>
             </li>
-            <li>
-              <Link to="/admin/courses">
-                <i className="fa fa-graduation-cap fa-fw" /> Courses
-              </Link>
-            </li>
-            <li>
-              <Link to="/admin/lessons">
-                <i className="fa fa-book fa-fw" /> Lessons
-              </Link>
-            </li>
-            <li>
-              <Link to="/dashboard">
-                <i className="fa fa-sitemap fa-fw" /> Organisations
-              </Link>
-            </li>
-            <li>
-              <Link to="/admin/users">
-                <i className="fa fa-users fa-fw" /> Users
-              </Link>
-            </li>
+            {this.state.admin ? (
+              <li>
+                <Link to="/admin/courses">
+                  <i className="fa fa-graduation-cap fa-fw" /> Courses
+                </Link>
+              </li>
+            ) : null}
+            {this.state.admin ? (
+              <li>
+                <Link to="/admin/lessons">
+                  <i className="fa fa-book fa-fw" /> Lessons
+                </Link>
+              </li>
+            ) : null}
+            {this.state.admin ? (
+              <li>
+                <Link to="/dashboard">
+                  <i className="fa fa-sitemap fa-fw" /> Organisations
+                </Link>
+              </li>
+            ) : null}
+            {this.state.admin ? (
+              <li>
+                <Link to="/admin/users">
+                  <i className="fa fa-users fa-fw" /> Users
+                </Link>
+              </li>
+            ) : null}
```

Comment:
Instead of having a condition about this.state.admin 4 times for each menu items, can you just do the condition once and group all the menu items together? So you have less duplicated code.

Reply:
To be clear, instead of :
`{this.state.admin ? <li> ...first item... </li> : null } {this.state.admin ? <li> ... second item... </li> : null }`
you can do:
`{this.state.admin ? <li> ... first item ...</li><li> ... second item ...</li> : null }`

Reply:
when I do that coming to this error "Syntax error: Adjacent JSX elements must be wrapped in an enclosing tag (55:14)"

Reply:
when I wrap it in any tag  it looks diffirent

Reply:
I think you can wrap it in an array, something like this:
`[ <li>...</li><li> ...</li>]`